hub/templates/hub/area.html

```diff
@@ -637,6 +636,33 @@ <h3 class="h5">{{ dataset.label }}</h3>
                     </div>
                 </div>
               {% endfor %}
+
+                <!-- TODO: Remove this after demo -->
```

Comment:
in theory we could stick this in an evaw specific template but probably not worth it for this demo :)